api/doc/openapi.yaml

```diff
@@ -557,72 +514,80 @@ components:
           type: string
         diem_ledger_version:
           $ref: '#/components/schemas/LedgerVersion'
+    Uint64:
+      title: uint64
+      type: string
+      format: uint64
+      description: Unsiged int64 type value
+      example: "32425224034"
     Address:
+      title: Account Address
```

Comment:
add title to all schemas (types), it shows up in the sidebar menu when spec is rendered into HTML